global-test.properties

```diff
@@ -19,8 +19,13 @@ intersmash.wildfly.operators.package_manifest=wildfly
 intersmash.wildfly.operators.channel=alpha
 
 # Wildfly images
-intersmash.wildfly.image=quay.io/wildfly/wildfly-s2i-jdk17
-intersmash.wildfly.runtime.image=quay.io/wildfly/wildfly-runtime-jdk17
+intersmash.wildfly.image=quay.io/wildfly/wildfly-s2i-jdk17:latest
+intersmash.wildfly.runtime.image=quay.io/wildfly/wildfly-runtime-jdk17:latest
+
+# Wildfly Helm Charts
+intersmash.wildfly.helm.charts.repo=https://github.com/wildfly/wildfly-charts.git
+intersmash.wildfly.helm.charts.branch=main
```

Comment:
@fabiobrz just upstream latest? why not some tag for stability's sake?

Reply:
You might be right. I'll ask